clientside_0_simple.py

```diff
@@ -0,0 +1,36 @@
+import dash
+print(dash.__version__)
+
+from dash.dependencies import Input, Output, State, ClientFunction
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+
+app = dash.Dash(__name__)
+
+app.layout = html.Div([
+    dcc.Input(id='input', value='hello world'),
+    html.Div(id='output-clientside'),
+    html.Div(id='output-serverside')
+])
+
+
+@app.callback(
+    Output('output-serverside', 'children'),
+    [Input('input', 'value')])
+def update_output(value):
+    return 'Server says "{}"'.format(value)
+
+
+app.callback(
```

Comment:
Code in the Dash review seems misaligned with usage here

- `clientside_function` https://github.com/plotly/dash/pull/672/files#diff-228c9aefac729977642672aa1416bacaR939
- `ClientsideFunction` vs `ClientFunction` https://github.com/plotly/dash/pull/672/files#diff-1b30af980419a25a5ffd5fdfc4ef0173R38